runtime.go

```diff
@@ -471,13 +471,33 @@ type SafeURL string
 
 // Script handling.
 
-// SafeScript encodes unknown parameters for safety.
-func SafeScript(functionName string, params ...interface{}) string {
+func safeEncodeScriptParams(escapeHtml bool, params ...interface{}) []string {
```

Comment:
This can lose the `...` and the `interface{}` since we don't need variadics here, and we're now supporting a later version of Go where we can use `any` instead.

```suggestion
func safeEncodeScriptParams(escapeHtml bool, params []any) []string {
```

Reply:
I wasn't able to drop the `...` but was able to replace `interface{}`. Switching to a slice causes tons of errors across the codebase.